src/api/config/routes/webui_routes.rb

```diff
@@ -227,7 +227,6 @@
         "/project/monitor/#{request.query_parameters[:project]}#{url_string}"
       }, constraints: ->(request) { request.query_parameters['project'].present? }
       # TODO: this should be POST (and the link AJAX)
```

Comment:
Is this comment still relevant?

Reply:
Oh, sorry! I didn't see your comment before merging. 
I'll remove it in another PR.